www/views.py

```diff
@@ -127,32 +138,33 @@ def gen_page_links(total, cur_page, pagesize, extra_params):
                 "title": str(i),
                 "url": "?page=" + str(i) + extra_params,
                 "class": "active",
-                "url": "#"
```

Comment:
I think the original idea and existing behavior is "#" but changing it to a link should be fine.

Reply:
You can see the `url` key being defined twice here and we need only one. If you think it's better keep `#` I can change.